### DISCUSSION TITLE
Yellow Color

### DISCUSSION THREAD

**Discussion by jcgmalave** [filamentphp/filament#2012] [Help]
How can I change the default yellow color of all interfaces?

**Comment by danharrin** [comment]
https://filamentphp.com/docs/2.x/admin/appearance#building-themes

**Reply by jcgmalave** [reply]
Is there any way to force only the yellow color, without creating a new theme?

**Reply by nettsite** [reply]
I did it today, for the first time - that yellow is a bit rough! 

It only takes 5 minutes, not much work at all.

**Reply by jcgmalave** [reply]
How did you do it my friend?

**Reply by ryangjchandler** [reply]
It's all here in the docs. https://filamentphp.com/docs/2.x/admin/appearance#building-themes

**Reply by studiowizjo** [reply]
To be extra clear:

1. Change in **tailwind.config.js** this: ```primary: colors.yellow``` to any other color
2. Run ```npm run watch``` in console to recompile your assets

**Reply by jcgmalave** [reply]
thanks a lot dear
